webapi/src/main/scala/org/knora/webapi/responders/v2/ontology/OntologyHelpers.scala

```diff
@@ -1560,18 +1464,72 @@ object OntologyHelpers {
   }
 }
 
-final case class OntologyHelpersLive(
-  triplestore: TriplestoreService,
-  ontologyCache: OntologyCache,
-)(implicit val stringFormatter: StringFormatter)
-    extends OntologyHelpers {
+trait OntologyTriplestoreHelpers {
```

Comment:
I think it would be more convenient for searching if they were in a separate file.

Reply:
seems reasonable, thx

Reply:
done: https://github.com/dasch-swiss/dsp-api/pull/3270/commits/4a3438d439018bc764495bb62e6c6c88cf43677d

Reply:
Amazing.